salt/modules/virtualenv_mod.py

```diff
@@ -82,8 +82,6 @@ def create(
     never_download=None,
     prompt=None,
     pip=False,
-    symlinks=None,
-    upgrade=None,
```

Comment:
I think these need to go through a deprecation process. You can't just remove parameters.